World/TestWorld/TestWorld.cpp

```diff
@@ -49,7 +49,7 @@ TestWorld::TestWorld(std::shared_ptr<ParametersTable> PT_)
 
 void TestWorld::evaluateSolo(std::shared_ptr<Organism> org, int analyze,
                              int visualize, int debug) {
-  auto brain = org->brains[brainNamePL->get(PT)];
+  auto brain = org->brain;
```

Comment:
I don't see the reason for this bringing back the deprecated org->brain, we should be using org->brains[ <brain name> ]

Reply:
You mentioned this is making a copy of a brain, but it's not.

Reply:
It must be making a copy of some sort. Try changing this line in feature-Analysis, and the recordHistory member value gets modified.

Reply:
With the PR #193 this solution won't work. This still leaves the question of how to get the actual brain out. My guess is that somewhere in Organism, the shared_ptr to the brain is having its underlying storage modified. I'll look into it. Let me know if you figure it out. Even more importantly, let me know if you don't spot the bug I'm talking about.